### PR TITLE
Assignment week 4

### PR DESCRIPTION
**Pull request by tungphamwd**

### CODE REVIEW

File: 02-express-tutorial/controllers/people.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  const index = people.findIndex((p) => p.id === id);
+
+  if (index === -1) {
+    return res
+      .status(404)
+      .json({ success: false, message: "Person not found" });
+  }
+
+  const updatedPeople = people.filter((person)=> {
+    return parseInt(person.id) !== id
```
Comment (joseluiz) [inline 02-express-tutorial/controllers/people.js]: Although the filter method is implemented correctly here, you are not removing the people from the people array. If we call the get endpoint, we can see that the people are still there.
To fix this, you can call the method splice to delete the people from the people array passing the index of person in the people array.
Here is an example of how you can do that:
people.splice(index, 1);

File: 02-express-tutorial/controllers/people.js
Comment (joseluiz) [inline 02-express-tutorial/controllers/people.js:1]: You forgot to add a method to update the person, add this method here and add a route using the router.put method.

**Review by joseluiz — CHANGES_REQUESTED** [review]
Hi Tung. You completed almost everything required for the lesson, you just forgot to add a method to update a person. Please see my comments on PR for details. If you have any questions you can call me on Slack.

File: 02-express-tutorial/controllers/people.js (hunk 1 above)
Comment (tungphamwd) [reply 02-express-tutorial/controllers/people.js]: Thank you. I already changed the code. Please check it again

File: 02-express-tutorial/controllers/people.js
Comment (tungphamwd) [reply 02-express-tutorial/controllers/people.js:1]: Thank you for your review. I added it. Please check it again.

**Review by joseluiz — APPROVED** [review]
PR approved.

**Pull request merged**